runtime/tr.source/trj9/optimizer/SPMDParallelizer.cpp

```diff
@@ -2641,48 +2641,33 @@ void TR_SPMDKernelParallelizer::insertGPUTemporariesLivenessCode(List<TR::TreeTo
 
    for (TR::TreeTop *insertionPoint = lit.getFirst(); insertionPoint; insertionPoint = lit.getNext())
       {
-      int i = 0;
-      TR::Node* currentNode;
-      TR::Node* prevNode;
+      TR::Node* currentNode = 0;
```

Comment:
`currentNode` is a pointer type. We should try to use `NULL` instead of 0 (until we finally support `nullptr` 🤞 )

Reply:
I can fix that.

Reply:
My updated change changed the 0 to a NULL.